specs-go/config.go

```diff
@@ -44,8 +44,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
-	// Capabilities are Linux capabilities that are kept for the container.
-	Capabilities []string `json:"capabilities,omitempty" platform:"linux"`
+	// Capabilities are Linux capabilities that are kept for the process.
+	Capabilities *LinuxCapabilities `json:"capabilities,omitempty" platform:"linux"`
```

Comment:
@jlbutler just removed the Linux-specific-ness of `process.capabilities` in #673 (at least [in the Markdown spec][1]).  With this change, it becomes even less clear to me how other OSes are supposed to configure their capabilities (although I [wasn't clear on that since #673][2], so it's not a lot worse ;).

[1]: https://github.com/opencontainers/runtime-spec/pull/673/files#diff-c9c91c29b41257aea3a3403cc606ad99L132
[2]: https://github.com/opencontainers/runtime-spec/pull/673/files#r99353136